### PR TITLE
Set up comment class for SA

### PR DESCRIPTION
**Pull request by mollygraton**
The second library used is Sentiment Analysis. This includes using gson to create a json comment object and then displaying the SA score in addition to the inputted comment.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -36,9 +41,11 @@
 public class DataServlet extends HttpServlet {
 
   private int maxNumOfComments = 3;
-  static final String ENTITY_NAME = "Input";
+  private float sentimentScore = 0;
```
Comment (cisxingg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Is this used?

**Review by cisxingg — APPROVED** [review]
One comment, but otherwise looks good :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (mollygraton) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: No, will delete

**Review by warfa-google — APPROVED** [review]
LGTM. Maybe something interesting would be to show an emoji based on the sentiment score instead of just printing it out.

**Pull request merged**